app/src/main/java/com/doubtless/doubtless/screens/search/SearchFragment.kt

```diff
@@ -85,7 +85,13 @@ class SearchFragment : Fragment() {
 
             searchJob = CoroutineScope(Dispatchers.Main).launch {
 
-                if (it.toString().length <= 4) return@launch
+                //if (it.toString().length <= 4) return@launch
+                //I feel its useless because there were problems with smaller keywords,
```

Comment:
you can remove the whole comment.

Reply:
bhaiya jaise 3 letter waaala keyword krna ho to like - dsa